### PR TITLE
Completed Word Count Exercism

### PR DESCRIPTION
**Pull request by judan**

### CODE REVIEW

File: word-count/word_count.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Phrase
+  attr_reader :words, :word_counts
```
Comment (MattRice12) [inline word-count/word_count.rb]: :word_counts is unused

File: word-count/word_count.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  def initialize(words)
+    @words = words
+  end
+
+  def word_count
+    words.downcase.scan(/\b[\w']+\b/).each_with_object(Hash.new(0)) { |word, hash| hash[word] += 1 }
```
Comment (MattRice12) [inline word-count/word_count.rb]: The regex can probably be assigned to a constant with a better name `ALL_WORD_CHARACTERS`

File: word-count/word_count.rb (hunk 1 above)
Comment (judan) [reply word-count/word_count.rb]: Removed!

File: word-count/word_count.rb (hunk 2 above)
Comment (judan) [reply word-count/word_count.rb]: Added a CONST for regex

**Pull request merged**